server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserMailboxesRoutes.java

```diff
@@ -280,7 +280,7 @@ public void defineMailboxExists() {
                     throw ErrorResponder.builder()
                         .statusCode(HttpStatus.NOT_FOUND_404)
                         .type(ErrorType.INVALID_ARGUMENT)
-                        .message("Invalid get on user mailboxes")
+                        .message("Mailbox does not exist")
```

Comment:
Why? And why removing the `details` finally?

Reply:
Because this case are clear enough without detail message.